kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ClusterCondition.java

```diff
@@ -26,6 +26,11 @@ static ClusterCondition filterNotExists(String cluster, String filterName) {
                 String.format("Filter \"%s\" does not exist.", filterName));
     }
 
+    static ClusterCondition ingressNotAvailable(String cluster, String ingressName) {
```

Comment:
TODO: use this somewhere to flag clusters with an ingress missing. (should we reconcile a VirtualCluster with a missing ingress, and omit that ingress, or omit the entire virtual cluster?)

Reply:
Let's keep things simple - let's just omit the entire virtual cluster as we are already doing for Filters.  If we want to relax that later, we can.

Reply:
will do, and I'll defer it to another PR, this ones already quite mega